docs/getting-started/integrate-auth/10_expressjs.mdx

```diff
@@ -65,7 +65,7 @@ Now that the Ory CLI is installed, create a new npm script to run the proxy serv
   "private": true,
   "scripts": {
     "dev": "nodemon index.js",
-    "proxy": "ory proxy --project $ORY_PROJECT_ID --port 4000 http://localhost:3000/"
+    "tunnel": "ory tunnel --dev http://localhost:3000",
```

Comment:
see above I think `--project $ORY_PROJECT_ID` is still needed? did not test this though